lib/node_modules/@stdlib/array/fixed-endian-factory/lib/main.js

```diff
@@ -644,6 +644,38 @@ function factory( dtype ) { // eslint-disable-line max-lines-per-function, stdli
 		return new this.constructor( flag2byteOrder( this._isLE ), out );
 	});
 
+	/**
+	* Returns the index of the last element in an array for which a predicate function returns a truthy value.
+	*
+	* @name findLastIndex
+	* @memberof TypedArray.prototype
+	* @type {Function}
+	* @param {Function} predicate - predicate function
+	* @param {*} [thisArg] - predicate function execution context
+	* @throws {TypeError} `this` must be a typed array
+	* @throws {TypeError} first argument must be a function
+	* @returns {integer} index or -1
+	*/
+	setReadOnly( TypedArray.prototype, 'findLastIndex', function findLastIndex( predicate, thisArg ) {
+		var buf;
+		var len;
+		var i;
+		if ( !isTypedArray( this ) ) {
+			throw new TypeError( format( 'invalid invocation. `this` is not %s %s.', CHAR2ARTICLE[ dtype[0] ], CTOR_NAME ) );
+		}
+		if ( !isFunction( predicate ) ) {
+			throw new TypeError( format( 'invalid argument. First argument must be a function. Value: `%s`.', predicate ) );
+		}
+		buf = this._buffer;
+		len = this._length;
+		for ( i = len - 1; i >= 0; i-- ) {
```

Comment:
```suggestion
		var i;
		if ( !isTypedArray( this ) ) {
			throw new TypeError( format( 'invalid invocation. `this` is not %s %s.', CHAR2ARTICLE[ dtype[0] ], CTOR_NAME ) );
		}
		if ( !isFunction( predicate ) ) {
			throw new TypeError( format( 'invalid argument. First argument must be a function. Value: `%s`.', predicate ) );
		}
		buf = this._buffer;
		for ( i = this._length - 1; i >= 0; i-- ) {
```

The local variable is not necessary in this case, as we are only accessing `_length` once.